lib/galaxy/tools/data_fetch.py

```diff
@@ -496,7 +496,7 @@ def __init__(self, request, registry, working_directory, allow_failed_collection
         self.check_content = request.get("check_content", True)
         self.to_posix_lines = request.get("to_posix_lines", False)
         self.space_to_tab = request.get("space_to_tab", False)
-        self.auto_decompress = request.get("auto_decompress", False)
```

Comment:
Does this change what the API does by default? It looks like I wanted to make callers indicate this if they want the data to be mucked with - but I might have not done that right.

Reply:
It does, and I think I found a better way so we can avoid this.